style.css

```diff
@@ -0,0 +1,62 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  font-family: 'Inter', sans-serif;
+  color: #172b4d;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+}
+
+/* styling the header */
+header {
+  justify-content: space-between;
+  align-items: center;
+  margin: 24px;
```

Comment:
Kindly remove the margin from the top and bottom as shown in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988) since it is introducing a lot of white-space 🤗

You can replace it like this:
`margin: 0 24px;`